src/index.js

```diff
@@ -116,13 +123,26 @@ export default class extends three.Sprite {
     // paint text
     ctx.font = font; // Set font again after canvas is resized, as context properties are reset
     ctx.fillStyle = this.color;
+    ctx.lineWidth = this.strokeWidth;
+    ctx.strokeStyle = this.strokeColor;
     ctx.textBaseline = 'bottom';
-
-    lines.forEach((line, index) => ctx.fillText(
-      line,
-      (innerWidth - ctx.measureText(line).width) / 2,
-      (index + 1) * this.fontSize
-    ));
+    
+    const drawTextStroke = this.strokeWidth && this.strokeColor !== 'transparent';
+    
+    lines.forEach((line, index) => {
+      if (drawTextStroke) {
+        ctx.strokeText(
```

Comment:
Shouldn't we stroke **after** we fill?

Reply:
I don't believe so, that just overlaps the stroke on top of the fill meaning you can't see the intended fill colour.

Reply:
I suppose it depends on how thick your stroke is, but I can't think of a scenario where it would look better over top.